src/app/home/home.component.ts

```diff
@@ -5,9 +5,8 @@ import { limitParam, pageParam } from '../_core/constants/pagination.constants';
   selector: 'app-home',
   imports: [RouterModule],
   templateUrl: './home.component.html',
-  styleUrl: './home.component.css',
 })
 export class HomeComponent {
-  limitParam = limitParam;
-  pageParam = pageParam;
+  limitParam: number = limitParam;
```

Comment:
use typescript helpers like `readonly` when properties are readonly like here. It prevent other developers to do mistakes when they work on the same code as you. 

Here I believe you want this limitParam to be readonly